cloud/deployment/fromfile/fromfile.go

```diff
@@ -86,21 +86,7 @@ func CreateOrUpdate(inputFile, action string, client astro.Client, coreClient as
 		return err
 	}
 
-	if deployment.IsDeploymentHosted(formattedDeployment.Deployment.Configuration.DeploymentType) {
-		getSharedClusterParams := astrocore.GetSharedClusterParams{
-			Region:        formattedDeployment.Deployment.Configuration.Region,
-			CloudProvider: astrocore.GetSharedClusterParamsCloudProvider(formattedDeployment.Deployment.Configuration.CloudProvider),
-		}
-		response, err := coreClient.GetSharedClusterWithResponse(context.Background(), &getSharedClusterParams)
-		if err != nil {
```

Comment:
We dont need to look up for a standard cluster when hosted standard deployment is being updated